src/core2/graph.test.js

```diff
@@ -625,6 +625,104 @@ describe("graph", () => {
     });
   });
 
+  describe("mergeConservative", () => {
```

Comment:
It doesn't test the plugin behavior except implicitly (via not throwing an error from the plugin handler not being registered). Regardless of what the precise plugin behavior is, it would be good to test.

Reply:
Are you suggesting that I add a test like
```js
it("installs appropriate plugin handlers", () => {
  const plugins = [new Handler()];
  expect(Graph.mergeConservative(plugins, []).plugins()).toEqual(plugins);
});
```
? If so, this seems reasonable to me, and I’m happy to add it. If not,
I don’t understand the comment.

Reply:
Yep, that was what I had in mind.